app/javascript/components/pages/ApartmentProtectedIndex.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import { Card, CardImg, CardText, CardBody,
+  CardTitle, CardSubtitle, Button } from 'reactstrap';
+
+export default function ApartmentProtectedIndex(props) {
+  let{apartments} = props
+  console.log("apartments", apartments)
+  
+  // if(apartments.length > 0){
+  //   console.log("apartment id test:", apartments[0].user_id)}
```

Comment:
Let's just move this code to the readme for now and come back to it-- or delete it.